packages/dbadmin/src/tools/init-schema-without-history.ts

```diff
@@ -4,10 +4,12 @@ import {
     PROPERTIES_TABLE,
     REFERENCES_TABLE,
     RESERVED_IDS_TABLE,
-    METAPOINTERS_TABLE
+    METAPOINTERS_TABLE,
+    CURRENT_DATA,
+    REPO_VERSIONS, CURRENT_DATA_REPO_VERSION_KEY, CURRENT_DATA_REPO_CLIENT_ID_KEY, CURRENT_DATA_LIONWEB_VERSION_KEY
 } from "@lionweb/repository-common";
 
-export function initSchemaWithoutHistory(schema: string): string {
+export function initSchemaWithoutHistory(schema: string, lionWebVersion: string): string {
```

Comment:
Can `schema` be renamed to `schemaName`, as it's a bit confusing – to me, at least – now?